### ISSUE TITLE
Pre-trained agents

### ISSUE THREAD

**Issue by novinsh** [benelot/pybullet-gym#11]
It would be good to provide some guidelines or any consideration that might be needed to provide some pre-trained agents. I need that for a project of mine and I can use some help regarding how the pre-trained agents should be produced, the format and loading and etc.

If you have any suggestion please share.

**Comment by benelot** [comment]
Pretrained agents for now just follow a simple guideline. They should be able to perform well on the environment at hand and gain a stable reward. To tune my environments, I need pretrained agents which perform well on the original mujoco envs so that I can tune my replicas to work similarly.

**Comment by benelot** [comment]
I see that you are pretty quick at training agents. Would you be interested in training some agents on the original mujoco environments to help me tune my replicas here? I am happy about agents trained in any framework, for now I prefer tensorforce since I have it set up already, but if it is easier for you to train in another framework, that is fine.
